game/game_result.go

```diff
@@ -62,10 +62,14 @@ func (gs GameScores) Assign() {
 	minTopperSize := int(math.Ceil(float64(numSCs)/float64(len(gs))) + 1)
 	// Score per SC is 34 / number of SCs.
 	scorePerSC := 34.0 / float64(numSCs)
+	survivalPart := 66.0 / float64(survivors)
 	// Tribute is one for each SCs over minimum topper size.
 	tributePerSurvivor := 0.0
 	if topperSize > minTopperSize {
 		tributePerSurvivor = scorePerSC * float64(topperSize-minTopperSize)
+		if tributePerSurvivor > survivalPart {
```

Comment:
When this is corrected, the check if the survivor has more points than the tribute part is redundant and should be removed, right?

Reply:
I took the liberty of removing the redundant case, the tests passed and it made sense.

Reply:
Thanks!